vite.config.shared.ts

```diff
@@ -160,7 +160,7 @@ export const getApiProxies = (pathToRoot: string = '../../../.') => {
     : undefined
 
   // Add additional regions as they become available
-  const availableRegions = ['au', 'eu', 'us']
+  const availableRegions = ['au', 'eu', 'us', 'me']
```

Comment:
Let’s go ahead and add the India region as well
```suggestion
  const availableRegions = ['au', 'eu', 'us', 'me', ‘in’]
```

Reply:
Updated with adding 'in'